data/defaultSpeedLimitHasUrbanOrRural.yml

```diff
@@ -0,0 +1,249 @@
+# the road is a dual carriageway, but this is if rural or urban matters at all, even in combination.
+
+# source: https://wiki.openstreetmap.org/wiki/Default_speed_limits
+
+default: true
+AD: true
+AE: true
+AF: false
+AG: true
+AL: true
+AM: true
+AO: true
+AR: true
+AS: true
+AT: true
+AU: true
+AU-ACT: true
+AU-NSW: true
+AU-NT: true
+AU-QLD: true
+AU-SA: true
+AU-TA: true
+AU-VIC: true
+AU-WA: true
+AW: true
+AZ: true
+BA: true
+BB: false
+BE-BRU: true
+BE-VLG: true
+BE-WAL: true
+BF: true
+BG: true
+BH: true
+BI: true
+BJ: true
+BN: true
+BO: true
+BR: true
+BS: true
+BT: true
+BW: true
+BY: true
+CA-AB: true
+CA-BC: true
+CA-MB: true
+CA-NB: true
+CA-NL: true
+CA-NS: true
+CA-NT: true
+CA-NU: true
+CA-ON: true
+CA-PE: true
+CA-QC: true
+CA-SK: false
+CA-YT: true
+CD: true
+CF: true
+CG: true
+CH: true
+CI: true
+CL: true
+CM: true
+CN: true
+CO: true
+CR: true
+CU: true
+CV: true
+CW: true
+CY: true
+CZ: true
+DE: true
+DK: true
+DO: true
+EC: true
+EE: true
+EG: true
+ES: true
+FI: true
+FJ: true
+FM: false
+FM-KSA: true
+FM-PNI: false
+FM-TRK: true
+FM-YAP: true
+FR: true
+GA: true
+GB: false
+GB-SCT: false
+GD: true
+GE: true
+GG: false
+GH: true
+GQ: true
+GR: true
+GT: true
+GU: false
+GY: true
+HK: false
+HR: true
+HT: false
+HU: true
+IE: true
+IL: true
+IM: true
+IN: true
+IO: true
+IT: true
+JE: false
+JP: false
+KE: true
+KG: true
+KH: true
+KI: true
+KN: true
+KR: true
+KZ: true
+LI: true
+LR: true
+LS: true
+LT: true
+LU: true
+LV: true
+MC: false
+MD: true
+MH: true
+MK: true
+MM: true
+MN: true
+MO: false
+MS: true
+MT: true
+MU: false
+MW: true
+MX: true
+MZ: true
+NA: true
+NE: true
+NG: true
+NI: true
+NL: true
+NL-BQ1: true
+NL-BQ2: true
+NL-BQ3: true
+NO: true
+NP: true
+NR: false
+NZ: true
+PA: true
+PE: true
+PG: true
+PH: true
+PK: true
+PL: true
+PN: false
+PR: true
+PS: true
+PT: true
+PY: true
+RO: true
+RS: true
+RU: true
+RW: true
+SA: true
+SD: true
+SE: true
+SG: false
+SH: false
+SI: true
+SK: true
+SN: false
+SO: true
+SS: true
+SV: true
+SX: true
+TD: true
+TH: true
+TL: true
+TN: true
+TR: true
+TT: true
+TV: false
+TW: false
+TZ: true
+UA: true
+UG: true
+US-AK: true
+US-AL: true
+US-AR: true
+US-AZ: true
+US-CA: true
+US-CO: true
+US-CT: false
+US-DC: false
+US-DE: true
+US-FL: true
+US-GA: true
+US-HI: false
+US-IA: true
+US-ID: false
+US-IL: true
+US-IN: true
+US-KS: true
+US-KY: true
+US-LA: false
+US-MA: true
+US-MD: true
+US-ME: true
+US-MI: true
+US-MN: true
+US-MO: true
+US-MS: false
+US-MT: true
+US-NC: true
+US-ND: true
+US-NE: true
+US-NH: true
+US-NJ: true
+US-NM: true
+US-NV: false
+US-NY: false
+US-OH: true
+US-OK: true
+US-OR: true
+US-PA: true
+US-RI: true
+US-SC: true
+US-SD: true
+US-TN: true
+US-TX: true
+US-UT: true
+US-VA: true
+US-VT: false
+US-WA: true
+US-WI: true
+US-WV: true
+US-WY: true
+UY: true
+UZ: true
+VE: true
+VI: true
+VN: true
+WS: true
+XK: true
+ZA: true
+ZM: true
+#Transnistria: true

```

Comment:
`true` is default, so missing support here is not a problem for that occupied territory

Reply:
Also question if `true` is the best default. It doesn't matter much though since most countries are defined here.

Reply:
true is best default because actually, most countries do have this distinction. It is reasonable to assume that this is the default. Half of the countries that do not have this distinction are city-states and island (islet) states anyway.